src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -81,8 +93,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
         for ($index = $tokens->count() - 1; $index > 0; --$index) {
             $token = $tokens[$index];
 
-            if ($token->equals('(') && !$tokens[$index - 1]->isGivenKind(T_ARRAY)) {
-                $this->fixFunction($tokens, $index);
+            if ($token->equals('(') && !$tokens[$index - 1]->isGivenKind([T_ARRAY, T_WHITESPACE])) {
```

Comment:
Not sure what is special/diff between array with and without white space in front of it.
Didn't test, but I think there is an existing bug in the code which is dealt with here in way that does not cover all cases.
 maybe this should be something like;
 ```php
&& !$tokens->getPrevMeaningfulToken($index)->isGivenKind(T_ARRAY)
```

please add a test like
```php
<?php
$a = array/**/(  1);
$a = array/**/( 12,    7);
$a = array/***/(123,  7);
$a = array (        1,
2);
```

this case should _not_ be touched/fixed